prog/vm/nexus.rb

```diff
@@ -11,14 +11,17 @@ class Prog::Vm::Nexus < Prog::Base
 
   def self.assemble(public_key, project_id, name: nil, size: "standard-2",
     unix_user: "ubi", location: "hetzner-hel1", boot_image: "ubuntu-jammy",
-    private_subnet_id: nil, nic_id: nil, storage_size_gib: 20, storage_encrypted: true,
+    private_subnet_id: nil, nic_id: nil, storage_size_gib: nil, storage_encrypted: true,
     enable_ip4: false)
 
     project = Project[project_id]
     unless project || Config.development?
       fail "No existing project"
     end
     Validation.validate_location(location, project&.provider)
+    vm_size = Validation.validate_vm_size(size)
+
+    storage_size = storage_size_gib || vm_size.storage_size_gib
```

Comment:
No need to define another variable. How about;

```ruby
storage_size_gib ||= vm_size.storage_size_gib
```

Reply:
good nit